src/mantarray_desktop_app/mc_simulator.py

```diff
@@ -71,6 +77,15 @@ def _get_secs_since_reboot_command(last_time: float) -> float:
     return perf_counter() - last_time
 
 
+def _get_secs_since_boot_up(start_time: float) -> float:
+    return perf_counter() - start_time
+
+
+def _get_secs_since_last_comm_from_pc(last_time: float) -> float:
+    return perf_counter() - last_time
+
+
+# pylint: disable=too-many-instance-attributes
```

Comment:
note for future: typically we provide some comment as a justification of why you think this situation warrants disabling one of the normal pylint rules

Reply:
also, I think (not 100% sure) that if you put the comment up here, it disables it for the rest of the module.  If you want to disable it just for a class, I think you can put it right inside the class declaration (i.e. line 90)

Reply:
It looks like that's how we are disabling it in all of the other subprocesses